pom.data.xml

```diff
@@ -543,7 +543,7 @@
   <modules>
     <module>./sdk/batch/microsoft-azure-batch</module>
     <module>./eventhubs/data-plane</module>
-    <module>./keyvault/data-plane</module>
+    <module>./sdk/keyvault</module>
```

Comment:
I know I'm a broken record here :-) but what is going to happen when track two key vault is moved in? This refers to the track one keyvault pom.xml parent file, but what is the plan for the track two keyvault pom.xml parent file?

Reply:
Should we replicate what we've done in the root of the repo and have a pom file for track 1 and one for track 2? If possible I'd like to keep both track 1 and track 2 libraries buildable for some time until track 2 has some mileage and we don't think we'll need to service track 1 anymore (could be years).

So we do need to have a solution that works for a while.

Reply:
We should set this to exactly path that contains the pom file the track1 library. We should try to avoid having a keyvault parent pom but in cases were we already have those and we need to keep them then we should rename them as data or client to mirror what we did in the root.

Reply:
What should be done about the readme.md file? We need to have a template for data-plane+client readme.md

Reply:
The Readme.md in the root service directory will be an overview of the service and a contain an index of all the directories/packages and explains what each is for and when to use which. That is the basic story that I chatted with @kaerm about doing for all our repo's. @kaerm is that work something that you were planning to tackle or is that something you were just planning to document?

Reply:
Correct, we need a README.md that will describe everything that resides there - T1, T2 and mgmt. I am happy to provide a template but will need some help from the teams working on each service as I do not know myself what's what